Modules/Sources/JetpackStats/Charts/BarChartView.swift

```diff
@@ -287,7 +287,12 @@ struct BarChartView: View {
     }
 
     private func getSelectedDataPoints(at location: CGPoint, proxy: ChartProxy, geometry: GeometryProxy) -> SelectedDataPoints? {
-        let origin = geometry[proxy.plotAreaFrame].origin
+
+        guard let frame = proxy.plotFrame else {
+            return nil
+        }
+
+        let origin = geometry[frame].origin
```

Comment:
This warning was in the same file so I addressed it – `plotAreaFrame` was renamed to `plotFrame` and made optional.